src/dotdom.js

```diff
@@ -37,7 +37,7 @@ module.exports = window;
    * @param {Array} [children] - The child VNode elements
    * @returns {VNode} Returns a virtual DOM instance
    */
-  let createElement = (element, props={}, ...children) => (
+  let c = (element, props={}, ...children) => (
```

Comment:
@wavesoft the long names were not getting minified by your minifier which is why I shortened them.